pyramid_oereb/standard/templates/plr_integer_primary_keys.py.mako

```diff
@@ -416,14 +416,14 @@ class PublicLawRestrictionDocument(Base):
     )
     document_id = sa.Column(
         sa.Integer,
-        sa.ForeignKey(DocumentBase.id),
+        sa.ForeignKey(Document.id),
         nullable=False
     )
     plr = relationship(
         PublicLawRestriction,
         backref='legal_provisions'
     )
     document = relationship(
-        DocumentBase
+        Document
```

Comment:
I think this relation drops too as there is no more link from DocumentBase to Document? Or am I'm wrong and it's the reference of a legal provision to the legal base relation?

Reply:
Yes good point !

Reply:
@kdeininger What do you think? I'am pro to @voisardf the relation from document to document is gone. So relation can be removed. Can you check this in detail?

Reply:
Yes, there are no relationships between documents anymore.

Reply:
Sorry, checked the wrong relationship. This relationships is at the PLR and is still needed. The class has changed from `DocumentBase` to `Document`. So everything should be fine.